src/pages/manage-my-registrations/delete/failure.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FunctionComponent } from "react";
+import { ReturnToYourAccountSection } from "../../../components/domain/ReturnToYourAccountSection";
+import { Grid } from "../../../components/Grid";
+import { Layout } from "../../../components/Layout";
+import { BeaconRegistryContactInfo } from "../../../components/Mca";
+import { SectionHeading } from "../../../components/Typography";
+
+const Success: FunctionComponent = (): JSX.Element => {
+  const pageHeading = "There was a problem deleting your registration";
+
+  return (
+    <>
+      <Layout
+        title={pageHeading}
+        pageHasErrors={false}
+        showCookieBanner={false}
+      >
+        <Grid
+          mainContent={
+            <>
+              <SectionHeading>
+                Please get in touch with the Beacon Registry Team
+              </SectionHeading>
+              <BeaconRegistryContactInfo />
+              <ReturnToYourAccountSection />
+            </>
+          }
+        />
+      </Layout>
+    </>
+  );
+};
```

Comment:
FYI, the unhappy path isn't tested in Cypress because we have no way of simply mocking a failed `deleteRegistration` (to my knowledge!).  Thoughts?